### PR TITLE
Nav drawer

### PR DESCRIPTION
**Pull request by ThieryMichel**
follow #758

### CODE REVIEW

File: src/app/js/public/Drawer.js [hunk 1]
```diff
@@ -22,126 +21,71 @@ const styles = StyleSheet.create({
...
+        )}
+    >
+        <div
+            className={classnames(
+                'drawer',
+                css(styles.drawer),
+                css(styles[status]),
+            )}
+            style={{
+                transition: `transform ${animationDuration}ms`,
```
Comment (Kmaschta) [inline src/app/js/public/Drawer.js:70]: Why this style isn't in the `drawer` class?

File: src/app/js/public/graph/GraphSummary.js [hunk 2]
```diff
@@ -31,54 +22,45 @@ const styles = {
...
-        >
-            <MenuItem
-                value=""
-                containerElement={<Link style={styles.link} to="/" />}
-                primaryText={polyglot.t('home')}
-                leftIcon={<HomeIcon />}
-            />
-            <MenuItem
+const PureGraphSummary = ({ graphFields, closeDrawer, p: polyglot }) => (
+    <div>
```
Comment (Kmaschta) [inline src/app/js/public/graph/GraphSummary.js:26]: Is this div mandatory?

File: src/app/js/public/Drawer.js (hunk 1 above)
Comment (ThieryMichel) [reply src/app/js/public/Drawer.js:70]: to have acces to the animationDuration props

File: src/app/js/public/graph/GraphSummary.js (hunk 2 above)
Comment (ThieryMichel) [reply src/app/js/public/graph/GraphSummary.js:26]: Maybe, maybe not ¯\\_(ツ)_/¯, the component will be totally redone in the next pr anyway

**Pull request merged**